clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  Heading,
+  HStack,
+  Link,
+  Stack,
+} from "@fidesui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import * as Yup from "yup";
+
+import { CustomSelect, CustomTextInput } from "../common/form/inputs";
+import {
+  AWS_REGION_OPTIONS,
+  DOCS_URL_AWS_PERMISSIONS,
+  DOCS_URL_IAM_POLICY,
+} from "./constants";
+import { useGenerateMutation } from "./scanner.slice";
+import { ScannerGenerateResponse } from "./types";
+
+// TODO: There should just be a theme/variant for blue links.
+const DocsLink = (props: React.ComponentProps<typeof Link>) => (
+  <Link isExternal color="complimentary.500" {...props} />
+);
+
+const initialValues = {
+  aws_access_key_id: "",
+  aws_secret_access_key: "",
+  region_name: "",
+};
+
+type FormValues = typeof initialValues;
```

Comment:
wow, I didn't know you could do this!

Reply:
Yeah I love some type inference from default/example values.